app/controllers/results_controller.rb

```diff
@@ -6,16 +6,26 @@ def create
     redirect_to result_path(assessment_code:)
   end
 
+  def early_result
```

Comment:
perhaps this should be display_early_result or show_early_result

Reply:
Because this method is a redirect on to the `def show`, I'm not sure about those names. This is kind of replicating what the full result method does above, where it intercepts to send off the CFE call and alter the session_data, then sends on to `def show`. So `display` and `show` might not be quite right - what about something like `def early_result_redirect`?

Reply:
yes that makes more sense